contracts/libraries/DataTypes.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.12;
+
+library DataTypes {
+    enum Modules {
```

Comment:
Rather than using an Enum, is we refactor modules to not be inherited on the Ops contract itself, we could use addresses instead, i.e. we pass an array of addresses of external modules to the `createTask` function (need a whitelist for that).